README.md

```diff
@@ -1,87 +1,125 @@
-# heat-docker-kubernetes
+# Getting started with k8s-docker-provisioner
 
-Steps:
+This guide will take you through the steps of deploying Kubernetes to Openstack using docker. The primary concept is described here: [docker-multinode](https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/kubernetes/blob/master/docs/getting-started-guides/docker-multinode/master.md)
 
-1. Deploy a Kubernetes cluster on DevStack using Heat
+This guide assumes you have a working OpenStack cluster.
 
+## Pre-Requisites
 
-## Directory layout
 
-Clone  <https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/devstack-vagrant.git>
+### Install OpenStack CLI tools
 
-* `~/your_workspace/devstack-vagrant/`
+- heat >= 0.9.0
+- nova >= 3.2.0
+```
+ sudo pip install -U python-heatclient
 
-## Setup DevStack
+ sudo pip install -U python-novaclient
+```
 
-Provision DevStack.
 
-```
-cd ~/your_workspace/devstack-vagrant/
-DEVSTACK_MEM=10024 DEVSTACK_CPUS=6 vagrant up --provider libvirt
-```
+### Configure Openstack CLI tools
+
+ Please get your OpenStack credential and modify the variables in the following files:
 
-Download custom image from Google drive: <https://drive.google.com/file/d/0B6KOuPCy8tK1aDRCaE16N1hkcUU/view>
-Uncompress it and upload to vgrant host
+ - **config-default.sh** Sets all parameters needed for heat template.
+ - **openrc-default.sh** Sets environment variables for communicating to OpenStack. These are consumed by the cli tools (heat, nova).
 
+### Get kubectl
+
+If you already have the kubectl, you can skip this step.
+
+kubectl is a command-line program for interacting with the Kubernetes API. The following steps should be done from a local workstation to get kubectl.
+Download kubectl from the Kubernetes release artifact site with the curl tool.
+
+The linux kubectl binary can be fetched with a command like:
 ```
-unxz centos7-docker.qcow2.xz
-scp -i devstack/.vagrant/machines/devstack/libvirt/private_key centos7-docker.qcow2 vagrant@192.168.123.100:/tmp/centos7-docker.qcow2
+$ curl -O https://storage.googleapis.com/kubernetes-release/release/v1.2.0/bin/linux/amd64/kubectl
 ```
-Upload custom Centos image to OpenStack.
 
+Make kubectl visible in your system.
 ```
-cd ~/your_workspace/devstack-vagrant/
-vagrant ssh
-
-sudo su - stack
-. devstack/openrc admin admin
-
-glance image-create --name centos7 --disk-format qcow2 --container-format bare --file /tmp/centos7-docker.qcow2 --is-public True
+sudo cp kubectl /usr/local/bin
 ```
 
-Create and register a keypair.
+### Prepare Openstack image
+
+The provisioning works on any operating system that has a Docker >= 1.10 and Docker-bootstrap service installed. This service is used to
+run flannel network inside of Docker containers themselves. Service should execute following command:
 
 ```
-ssh-keygen -f ~/.ssh/id_rsa -P ''
-nova keypair-add keypair1 --pub-key ~/.ssh/id_rsa.pub
+docker daemon \
```

Comment:
This is confusing. The reader will try to execute the docker command. I suggest to remove and leave a TODO.

Later, I would suggest a separate document that describes the docker bootstrap concept and the manual steps. Maybe this document should be part of the other project.